src/js/themes/base.d.ts

```diff
@@ -1087,7 +1088,11 @@ export interface ThemeType {
   };
   menu?: {
     background?: BackgroundType;
-    item?: ButtonType;
+    item?:
+      | ButtonType
+      | {
+          align?: AlignType;
+        };
```

Comment:
I think we should allow people to pass button props in addition to align rather than only allowing passing one or the other.

Reply:
oh right that shouldnt be OR ill fix